sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/HttpPostControllerActionShouldValidateInput.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
```

Comment:
A general question about known/expected false positives: if we can think of cases that would cause false positives, is it worth calling them out explicitly by writing a test case and commenting that's it's a "known" false positive?

An example here would be code with a custom attribute with a similar name that isn't from the core ASP.Net framework attribute e.g. _HttpPostHandler_.

Reply:
I think that's better to put as many cases as possible. I'll update my implementation to work with symbols for now on and we will see later (we will have to update a lot of rules anyway).